.github/workflows/ci.yaml

```diff
@@ -18,6 +18,31 @@ jobs:
           path: ~/.cache/pip
           key: ${{ hashFiles('pyproject.toml') }}
       - uses: pre-commit/action@v2.0.3
+  Mypy:
+    runs-on: ubuntu-latest
+    name: Mypy
+    env:
+      MYPY_FORCE_COLOR: 1
+      TERM: xterm-color
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/pip
+          key: mypy-${{ runner.os }}-${{ hashFiles('pyproject.toml') }}
+      - name: Install Dependencies
+        run: |
+          pip install wheel
+          pip install -U -e .[all,mypy]
```

Comment:
@kaxil what is the befit of installing `mypy` like this as opposed to adding to the `tests` section of the `pyproject.yml` file?

Reply:
I did that mainly because once we start adding more modules we will have more "Typing" dependencies, for example:

https://github.com/astronomer/astronomer-providers/blob/main/setup.cfg#L81-L103

```
mypy =
    mypy>=0.800
    types-aiofiles
    types-boto
    types-certifi
    types-croniter
    types-Deprecated
    types-docutils
    types-freezegun
    types-paramiko
    types-protobuf
    types-python-dateutil
    types-python-slugify
    types-pytz
    types-redis
    types-requests
    types-setuptools
    types-termcolor
    types-tabulate
    types-toml
    types-Markdown
    types-PyMySQL
    types-PyYAML
```